CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#849](https://github.com/nf-core/ampliseq/pull/849) - Added Greengenes version 2024.09 of DADA2 taxonomy database: `greengenes2=2024.09` or `greengenes2` as parameter to `--dada2_ref_taxonomy`
 - [#853](https://github.com/nf-core/ampliseq/pull/853) - Added Greengenes2 version 2024.09 for taxonomic classification with QIIME2: `greengenes2=2024.09` or `greengenes2` as parameter to `--qiime_ref_taxonomy`
 - [#864](https://github.com/nf-core/ampliseq/pull/864) - Taxonomic assignment using DADA2 is done in chunks of 10,000 ASVs by default, the chunk size can be adjusted with `--dada_assign_chunksize`
+- [#879](https://github.com/nf-core/ampliseq/pull/879) - Add `sbdi-gtdb=R10-RS226-1` as parameter to `--dada_ref_taxonomy` and make this version the default for `sbdi-gtdb`.
```

Comment:
one section up please ;)

Reply:
One line up, or?

Reply:
I thought it is well placed header `nf-core/ampliseq version 2.14.0dev`, i.e. to line 9 or such. Currently it is under header `nf-core/ampliseq version 2.13.0 - 2025-04-04`. Not sure if I hallucinate :D

Reply:
Right, you're not hallucinating, but I'm apparently blind :-)